runtimerevolution/web3-roulette
# [WEB3-38] Create giveaway datetimes & UI updates

# Description

This PR changes the start/end date types from date to datetimes. It also includes some UI changes and refractor to the create/edit giveaway page, such as:
- Bold labels
- Conditions title and add condition button
- Delete icon
- New giveaway title
- Save / cancel buttons
- Page responsiveness

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-38

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Go to the create new giveaway page
- Verify the UI
- Verify that the start and end dates prompt date and time pickers
- Create a giveaway
- Verify that the start and end datetimes correspond to what was selected

## Checklist:
